kbase-extension/static/kbase/js/common/jobStateViewer.js

```diff
@@ -101,9 +101,11 @@ define([
                 this.lastJobState = this.jobManager.getJob(this.jobId);
                 this.renderJobState(this.lastJobState || { job_id: this.jobId });
 
-                this.bus.emit(jcm.MESSAGE_TYPE.STATUS, {
-                    [jcm.PARAM.JOB_ID]: this.jobId,
-                });
+                if (!this.lastJobState || !this.lastJobState.status) {
+                    this.bus.emit(jcm.MESSAGE_TYPE.STATUS, {
+                        [jcm.PARAM.JOB_ID]: this.jobId,
+                    });
```

Comment:
only request job status if there isn't a complete job state object (although that should all be handled by the job manager)